Mathlib/Data/Set/Accumulate.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2020 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+Ported by: Anatole Dedecker
+
+! This file was ported from Lean 3 source module data.set.accumulate
+! leanprover-community/mathlib commit 207cfac9fcd06138865b5d04f7091e46d9320432
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Set.Lattice
+
+/-!
+# Accumulate
+
+The function `Accumulate` takes a set `s` and returns `⋃ y ≤ x, s y`.
+-/
+
+
+variable {α β γ : Type _} {s : α → Set β} {t : α → Set γ}
+
+namespace Set
+
+/-- `Accumulate s` is the union of `s y` for `y ≤ x`. -/
+def Accumulate [LE α] (s : α → Set β) (x : α) : Set β :=
```

Comment:
Shouldn't this name not be capitalized?

Reply:
Indeed. Sounds like I wasn't familiar with the new naming conventions back then.